### PR TITLE
added destroyToken and logoutAndDestroyToken functions

### PR DESCRIPTION
**Pull request by amuramoto**

### CODE REVIEW

File: sdks/html5-javascript/lib/modules/Client.js [hunk 1]
```diff
@@ -682,6 +682,70 @@
...
+				options.endpoint = 'users/'+username+'/revoketoken?token='+this.getToken();
+			} else {
+				options.endpoint = 'users/'+username+'/revoketoken?token='+token;
+			}
+      this.request(options, function(err,data) {
+		    if (err) {
+	          if (self.logging) {
+	            console.log('error destroying access token');
+	          }
+	          doCallback(callback, [err, data, null], self);
```
Comment (r3b) [inline sdks/html5-javascript/lib/modules/Client.js:714]: You don't have 'callback' as a parameter, so this code would never be executed.

File: sdks/html5-javascript/lib/modules/Client.js [hunk 2]
```diff
@@ -682,6 +682,70 @@
...
+    *	 @param {string} token set to 'null' to revoke the token of the currently logged in user 
+    *  	 or set to token value to revoke a specific token
+    *	 @param {string} revokeAll set to 'true' to revoke all tokens for the user        
+    *  @return none
+    */
+    Usergrid.Client.prototype.logoutAndDestroyToken = function(username, token, revokeAll) {
+			if (username == null) {
+				console.log('username required to revoke tokens');
+			} else {
+				this.destroyToken(username,token,revokeAll);
```
Comment (r3b) [inline sdks/html5-javascript/lib/modules/Client.js]: Once you add the callback to the previous function, you should add it to this function and pass it through.

**Comment by rodsimpson** [comment]
Alex, do you agree to donate this code to the Apache Software Foundation?

**Comment by ghost** [comment]
Sure looks good! Merge away!

**Comment by amuramoto** [comment]
Aboslutely. Please merge.

**Comment by rodsimpson** [comment]
Thanks @amuramoto !!

**Pull request merged**